VERSION

```diff
@@ -1 +1 @@
-1!5.11.1
+1!5.12.1
```

Comment:
Nit: For future reference, when bumping minor version, patch version should be `.0` rather than `.1`. It doesn't hurt anything though and the previous versions should be fairly obvious, so I'm ok merging without changing it.